webui/react/src/services/decoder.ts

```diff
@@ -618,21 +618,30 @@ export const decodeV1TrialToTrialItem = (data: Sdk.Trialv1Trial): types.TrialIte
 };
 
 const decodeSummaryMetrics = (data: Sdk.V1SummarizedMetric[]): types.MetricContainer[] => {
-  return data.map((m) => ({
-    data: m.data.map((pt) => ({
-      batches: pt.batches,
-      value: pt.value,
-    })),
-    name: m.name,
-    time: m.time?.map((pt) => ({
-      time: pt.time,
-      value: pt.value,
-    })),
-    type:
-      m.type === Sdk.V1MetricType.TRAINING
-        ? types.MetricType.Training
-        : types.MetricType.Validation,
-  }));
+  return data.map((m) => {
+    const metrics: types.MetricContainer = {
+      data: m.data.map((pt) => ({
+        batches: pt.batches,
+        value: pt.value,
+      })),
+      name: m.name,
+      time: m.time?.map((pt) => ({
+        time: pt.time,
+        value: pt.value,
+      })),
+      type:
+        m.type === Sdk.V1MetricType.TRAINING
+          ? types.MetricType.Training
+          : types.MetricType.Validation,
+    };
+    if (m.epochs && m.epochs.length > 0) {
+      metrics.epochs = m.epochs.map((pt) => ({
+        epoch: pt.epoch,
+        value: pt.value,
+      }));
+    }
```

Comment:
can put `epochs : m.epochs?.map(...)` for consistency with with the way we are doing it for `time`?